class oop.py

```diff
@@ -0,0 +1,79 @@
+class Critter(object):
+    """ The virtual pet """
+    def __init__(self, name, hunger=0, boredom=0):
+        self.name = name
+        self.hunger = hunger
+        self.boredom = boredom
+
+    def __pass_time(self):
+        self.hunger += 1
+        self.boredom += 1
+
+    @property
+    def mood(self):
+        unhappiness = self.hunger + self.boredom
+        if unhappiness < 5:
+            m = 'happy'
+        elif 11 <= unhappiness <= 15:
+            m = 'frustrated'
+        else:
+            m = 'mad'
+        return m
```

Comment:
Can't you shorten this code to 
```suggestion
        if unhappiness < 5:
            return 'happy'
        elif 11 <= unhappiness <= 15:
            return 'frustrated'
        else:
            return 'mad'
```

Reply:
oh yea 👍

Reply:
You gonna make the changes?